pkgs/development/python-modules/aiocontextvars/default.nix

```diff
@@ -25,10 +24,6 @@ buildPythonPackage rec {
     pytest-runner
   ];
```

Comment:
```suggestion
```
pytest-runner is deprecated and does not work in the sandbox anyway

Reply:
Feel free to open a new PR to resolve this,